Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/robertbartlettbaron/hammerdb:latest
+FROM ghcr.io/cci-moc/hammerdb:master
```

Comment:
what is the difference between ghcr.io/cci-moc/hammerdb:master & ghcr.io/cci-moc/k-hammer:nightly

Reply:
Both use GitHub's container registry.  cci-mod is owned by the MOC, I placed it here as it is meant as a general tool and not specifically for the nerc.